src/test/java/org/apache/mesos/scheduler/recovery/RecoverySchedulerTest.java

```diff
@@ -41,7 +41,7 @@
  * as failed
  */
 public class RecoverySchedulerTest {
-    private DefaultRecoveryScheduler recoveryScheduler;
+    private DefaultRecoveryScheduler repairScheduler;
```

Comment:
This is just a mistake from before.  It used to be called a `DefaultRepairScheduler` but the variable name didn't keep up.